clientd3d/d3drender.c

```diff
@@ -13,8 +13,9 @@
 #define	TEX_CACHE_MAX_WALLMASK	2000000
 #define	TEX_CACHE_MAX_EFFECT	2000000
 #define	TEX_CACHE_MAX_PARTICLE	1000000
-#define FOV_H					((gD3DRect.right - gD3DRect.left) / (float)(MAXX * stretchfactor) * (-PI / 3.6f))
-#define FOV_V					((gD3DRect.bottom - gD3DRect.top) / (float)(MAXY * stretchfactor) * (PI / 6.0f))
+// Define field of views with magic numbers for tuning
+#define FOV_H					((gD3DRect.right - gD3DRect.left) / (float)(main_viewport_width) * (-PI/3.6f))

```

Comment:
Make this an inline function instead?  Macros are gross.

Reply:
new inline functions added